src/components/mixins/Leaderboard.js

```diff
@@ -56,17 +52,30 @@ module.exports = {
     });
   },
 
-  processPage: function(page) {
-    return {
-      id: page.id,
-      name: page.name,
-      url: page.url,
-      isoCode: page.amount.currency.iso_code,
-      amount:  page.amount.cents,
-      totalMembers: page.team_member_uids.length,
-      imgSrc: page.image.large_image_url,
-      medImgSrc: page.image.medium_image_url
-    };
+  CheckHasContentCallback: function(leaderboard) {
```

Comment:
@davidherse It is easier to test the `onHasContent` callback. Before I had to compose a leaderboard object to pass in to the processLeaderboard method. Now I only need an array `[1]`.